From c3b128736dbde629db87751cd706a0b68a41e7d0 Mon Sep 17 00:00:00 2001
From: Tom Gundersen <teg@jklm.no>
Date: Fri, 3 Oct 2014 08:43:34 +0200
Subject: [PATCH] glib-event-glue: remove some unnecessary lines

Not needed in an example. Should still shorten the license, but should make sure it is still complete so people can copy-paste without problems.
---
 man/glib-event-glue.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/man/glib-event-glue.c b/man/glib-event-glue.c
index 95aaea1e63..c3719e3378 100644
--- a/man/glib-event-glue.c
+++ b/man/glib-event-glue.c
@@ -1,5 +1,3 @@
-/*-*- Mode: C; c-basic-offset: 8; indent-tabs-mode: nil -*-*/
-
 /***
   Copyright 2014 Tom Gundersen
 
@@ -28,8 +26,6 @@
 #include <errno.h>
 #include <stdlib.h>
 
-#include "glib-event-glue.h"
-
 typedef struct SDEventSource {
         GSource source;
         GPollFD pollfd;
